modules/core/src/main/java/org/apache/ignite/cache/QueryEntity.java

```diff
@@ -103,6 +103,20 @@
     /** Scale for fields. */
     private Map<String, Integer> fieldsScale = new HashMap<>();
 
+    /**
+     * Used for composite primary key.
+     * {@code true} if the PK index is created on fields of PK;
```

Comment:
Let's use the same sign fin the end of every line for public docs: "." instead ";".